### PR TITLE
Add 1.8 headline features

### PR DESCRIPTION
**Pull request by domcleal**
@ares or @tstrachota, would you mind giving this a quick read to ensure I've described the networking changes fairly?

If there's anything you think I should highlight, but haven't, please let me know.  The full list of changes is in the main release notes section below.

### CODE REVIEW

File: _includes/manuals/1.8/1.2_release_notes.md [hunk 1]
```diff
@@ -2,7 +2,37 @@
...
+
+##### Unified networking UI
+
+The new host form now has a single user interface that combines all three of the networking aspects shown separately in previous versions - the primary interface, additional interfaces and virtual machine interfaces.  This makes it easier to configure complex networking setups, as all configured interfaces are shown in a single list and can be added, edited and deleted.
+
+When working with compute resources such as libvirt and VMware, additional interfaces are added to the virtual machine for every interface added on the Interfaces tab automatically, keeping it in sync.
+
+##### Provisioning interface support
+
+Traditionally, Foreman stored the primary interface and additional interfaces separately.  Foreman 1.8 now stores a single list of network interfaces for each host, one of which can be marked as the primary interface for the purposes of determining the FQDN and primary IP address.
```
Comment (ares) [inline _includes/manuals/1.8/1.2_release_notes.md:15]: maybe: one of which **must** be marked as the primary interface

**Comment by ares** [comment]
except one nit, :+1: for the networking part

**Comment by tstrachota** [comment]
@domcleal :+1: from my side (apart from the can/must)

**Comment by domcleal** [comment]
Thanks for the review, merged as d744ac0.

**Pull request closed without merging**